.dockerignore

```diff
@@ -0,0 +1,13 @@
+**/*.yml
+**/*.yaml
+**/*.md
+**/*example
+Dockerfile?
```

Comment:
Looks like the dockerfiles are still getting picked up:
```
root@304bbf971d5d:/# ls docker/
Dockerfile  Dockerfile-aio  Dockerfile_local  docker-entrypoint.sh
```
I suspect that if you wanted to ignore everywhere you'd need `**/Dockerfile*`

Reply:
Good catch. I updated it.